### ISSUE TITLE
Introductory Session to Chinese Orchestra

### ISSUE THREAD

**Issue by lowryan** [OpenSUTD/DiscoverSUTD-2021-microsite#93]
**Session Name**
Introductory Session to Chinese Orchestra

**Abstract**
An introductory session which aims to expose members of the SUTD community to the different musical instruments found in the Chinese Orchestra, as well as to have a glimpse of what we do in SUTDCO.

**Student Organisation organising this session**
SUTD Chinese Orchestra

**Event Format**
Physical

**Session intent (what to expect)**
Participants will learn about:
- The background and achievements of SUTDCO
- The structure of a Chinese Orchestra / Chinese Ensemble
- Activities to look forward to in SUTDCO
Participants will also get to try out different musical instruments (except for wind instruments)

**Session Details**
Venue: Music Room 1
Dates and Time of Sessions:
- 24 Sep (Fri) 3.00pm - 4.15pm
- 24 Sep (Fri) 4.30pm - 5.45pm
- 27 Sep (Mon) 6.00pm - 7.15pm
- 27 Sep (Mon) 7.30pm - 8.45pm
Total number of participants per session: 10 (3 facilitators + 7 participants)

**Speakers / Workshop facilitators**
The facilitators for the introductory sessions will be members of the SUTDCO Executive Committee, namely
- Low Ryan (President)
- Tan Kewen (Vice-President)
- Fu Meihui (Vice-President)
- Li Xueqing (Secretary)
- Crystal Chia Sze Yun (Quartermaster)
- Seah Wan Xuan (Treasurer)

**Sign up link (if any)**
https://forms.office.com/r/KsvSqJ6JLY

**Logistical details / Budget Needed**
Will need to use Music Room 1 for the duration of the sessions stated above (instruments are stored inside MR1).
Telegram group for new freshies: https://t.me/joinchat/WZVW_IQ0h6YzYzVl
Estimated Budget: ~$48 (for refreshments)
Breakdown: 
- 30x Milo Packets: $17.75
- 32x Rice Crackers: $3.80
- 27x Oreo Packets: $6.75
- 36x Yupi Gummy Candies: $4.05
- 30x Hello Panda: $15
- Total: $47.35

Please provide us with your club e-mail address so we can notify you once your event has been scheduled. 
Email: chineseorchestra@club.sutd.edu.sg

**Comment by ExtremelySunnyYK** [comment]
Hi @lowryan thanks for submitting the event proposal for Chinese Orchestra. 

Could we kindly have the budget breakdown of the refreshments planned for purchase?

Do note that claims would have to be made on concur with receipts.

**Comment by lowryan** [comment]
Hi, the following is the budget breakdown for the refreshments, we have estimated it to be around $47.35 (rounded up to $48 above):
- 30x Milo Packets: $17.75
- 32x Rice Crackers: $3.80
- 27x Oreo Packets: $6.75
- 36x Yupi Gummy Candies: $4.05
- 30x Hello Panda: $15
- Total: $47.35

Additionally, I have amended the dates for our sessions, it will now be held on 24 and 27 September, instead of 22 and 27 September previously.

Thank you!

**Comment by ExtremelySunnyYK** [comment]
Thank you for the update ryan!

**Comment by castlemas** [comment]
@lowryan 

Hi Ryan, I noted that you would not need help with the booking of the Music Room venue, **may I know if you'll be requiring access to the Music room?
 If so, could I have at most 2 facilitators' Student IDs and Email so that we could grant you card access to the Music Room?**

Thank you xx

**Comment by lowryan** [comment]
Yes, we would require card access to the Music Room for the sessions on both days.

Facilitator Details:
Low Ryan (1005257) - ryan_low@mymail.sutd.edu.sg
Tan Kewen (1004562) - kewen_tan@mymail.sutd.edu.sg

Thanks for your help!

**Comment by jolow99** [comment]
Dear @lowryan  Your event has been approved by the school admin. You have been allocated Music Room 1 on 24 Sep 3pm-6pm and 27 Sep 6pm - 9pm. The full $47.35 of requested budget has been approved as well. 

Wishing you a smooth and successful event!

**Comment by lowryan** [comment]
Hi DiscoverSUTD team,

Due to the cancellation of physical events from now till 24 Sep, our sessions on 24 Sep have been affected. As such, our club is planning to open up new dates for physical events the week after. If we were to hold physical sessions on say 29 Sep and 1 Oct, will the approval of the sessions be done through your team or will it have to be done through OSL (sending EPF?)

**Comment by lowryan** [comment]
Hi DiscoverSUTD team,

Our club is planning to reschedule 2 of our sessions (both previously happening on 24 Sep) to 1 Oct. Our sessions will remain physical and will be held in Music Room 1. I have updated the above issue to reflect the new session dates. Could I check if booking of the venue (Music Room 1) may be done through your team or through OSL? Thanks!

**Comment by makehahaz** [comment]
Hi @lowryan as per communicated through telegram. We will try to help your club for the 27 Sep booking.

**Comment by lowryan** [comment]
Alright, thanks for your help!
